### PR TITLE
Update Polish (pl) localization of AMO Frontend

### PR DESCRIPTION
**Pull request by piotrdrag**
needs-review: @splewako

### CODE REVIEW

File: locale/pl/LC_MESSAGES/disco.po [hunk 1]
```diff
@@ -15,91 +15,93 @@ msgstr ""
 
 #: src/disco/components/Addon.js:112
 msgid "Close"
-msgstr ""
+msgstr "Zamknij"
 
 #: src/disco/components/Addon.js:141
 msgid "Hover to preview or click to install %(name)s"
 msgstr ""
+"Najechanie wyświetli podgląd, kliknięcie zainstaluje dodatek „%(name)s”"
```
Comment (splewako) [inline locale/pl/LC_MESSAGES/disco.po]: How this string is used? Wouldn't it be better to translate it more like "Najedź, aby wyświetlić podgląd"?

File: locale/pl/LC_MESSAGES/disco.po [hunk 2]
```diff
@@ -15,91 +15,93 @@ msgstr ""
...
 
 #: src/disco/components/Addon.js:151
 msgid "Hover over the image to preview"
-msgstr ""
+msgstr "Najechanie na obraz wyświetli podgląd"
```
Comment (splewako) [inline locale/pl/LC_MESSAGES/disco.po]: Like above

File: locale/pl/LC_MESSAGES/disco.po (hunk 1 above)
Comment (piotrdrag) [inline locale/pl/LC_MESSAGES/disco.po]: Screenshot
Probably. I'm gonna change it.

Comment (splewako) [inline locale/pl/LC_MESSAGES/disco.po]: As you prefer, UI is ugly (in kind words).

File: locale/pl/LC_MESSAGES/disco.po [hunk 3]
```diff
@@ -15,91 +15,94 @@ msgstr ""
...
 
 #: src/disco/components/Addon.js:151
 msgid "Hover over the image to preview"
-msgstr ""
+msgstr "Najedź na obraz, aby wyświetlić podgląd"
 
 #: src/disco/components/Addon.js:166
 msgid "Installation failed. Please try again."
-msgstr ""
+msgstr "Instalacja się nie powiodła. Proszę spróbować ponownie."
```
Comment (splewako) [inline locale/pl/LC_MESSAGES/disco.po]: Personally I prefer "Instalacja nie powiodła się." (just a nit).

**Comment by splewako** [comment]
Otherwise looks good.

**Comment by piotrdrag** [comment]
Thank you!

**Comment by piotrdrag** [comment]
I would appreciate a merge. Our Polish users are still seeing an English page in their Firefoxes.

**Pull request merged**